INSTALL.md

```diff
@@ -93,6 +89,27 @@ Under Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
 $ sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp4-extra camlp5 proofgeneral proofgeneral-doc libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib
 ```
 
+Arch Linux and Manjaro Linux:
+
+    To prepare for compiling it yourself:
+
+      sudo pacman --sync --needed archlinux-keyring
+      sudo pacman-key --populate archlinux
+      sudo pacman --sync --needed ocaml camlp5 ocaml-findlib
+
+    Install ProofGeneral from its website as described above.  Or,
+    alternatively, install it from AUR (
+    https://aur.archlinux.org/packages/proofgeneral/ ) using an auxiliary
+    package manager such as "cower" ( https://aur.archlinux.org/packages/cower/
+    ) or yaourt ( https://aur.archlinux.org/packages/yaourt/ ).  That approach
+    will be especially convenient for expert Arch Linux users who find other
+    packages at AUR worth installing.
```

Comment:
Can we shorten this paragraph? We are not intending to give an introduction to the use of Arch. In particular, I don't feel we need to advertise package managers for Arch in the UniMath install manual.

Every non-essential information we give in this manual is counterproductive, as it makes the manual harder to use.